community-plugins.json

```diff
@@ -613,4 +613,16 @@
     },
     "deleted": false
   }
+  {
+    "id": "mun-assistant",
+    "name": "MUN Assistant",
+    "author": "AbeNaws",
+    "description": "An assistant to help summarize countries' positions at Model United Nations events",
+    "image": "/plugins/logos/mun-assistant.jpg",
```

Comment:
please check the image path (.png?) @AbeNaws , and, could you show me a real use-case of this plugin?

Reply:
Oops, you're right, that should be .png

As for the use case, I am part of a debate league, and we were holding a Model United Nations event, so I made this to help me. It's a really niche use case, but I was trying to get it in so that I could use it. If there is an easier way to add plugins for my personal use cases, that would be fantastic, but I couldn't find one.